backend/src/main/java/com/google/blackswan/mock/SimpleRelatedDataGenerator.java

```diff
@@ -81,17 +75,17 @@ public static SimpleRelatedDataGenerator createGenerator() {
   private void prefillRelatedData() {
     // TODO: Replace with call for querying configs from datastore.
     // TODO: Deal with capitalizations when querying config from datastore.
-    relatedDataMap.put(DataInfo.of(INTEREST_METRIC, "Ramen"), 
-        DataInfoUser.of(INTEREST_METRIC, "Udon", CONFIG_USERNAME));
-    relatedDataMap.put(DataInfo.of(INTEREST_METRIC, "Ramen"), 
-        DataInfoUser.of(INTEREST_METRIC, "Pho", CONFIG_USERNAME));
+    relatedDataMap.put(DataInfo.of(Constant.INTEREST_US, Constant.RAMEN), 
+        DataInfoUser.of(Constant.INTEREST_US, Constant.UDON, CONFIG_USERNAME));
+    relatedDataMap.put(DataInfo.of(Constant.INTEREST_US, Constant.RAMEN), 
+        DataInfoUser.of(Constant.INTEREST_US, Constant.PHO, CONFIG_USERNAME));
     // Multimap looks like this right now: 
     // {{Interest Level, Ramen},{{Interest Level, Udon}, {Interest Level, Pho}}}, ...}.
   }
 
   private ImmutableMap<Timestamp, Integer> getTopicDataPoints(DataInfo topic) {
     return ImmutableMap.copyOf(csvDataCache.computeIfAbsent(topic, key -> CSVParser.parseCSV(
-        SimpleRelatedDataGenerator.class.getResourceAsStream(FILE_LOCATIONS.get(key))
+        LocalFileSystem.createSystem().getDataAsStream(topic)
```

Comment:
Instead of using LocalFileSystem in code, you can define FileSystem as a class variable and default it to LocalFileSystem.
private static final FileSystem = LocalFileSystem.createSystem();

Ideally we want to inject FileSystem in constructor. Without injection, defining it at top is like calling out this assumption can be changed over time, and when injection is available you can easily switch it to injection.

Reply:
Yep, I changed it! Thanks for the suggestions. This will make future injection more obvious :) 

This will also help when I want to set a flag for which file system to use in a future PR.